.github/vale/styles/config/vocabularies/FusionAuth/accept.txt

```diff
@@ -129,3 +129,8 @@ xkcd
 Yubico
 YubiKey
 Zendesk
+Mailgun
+Sinch
+Brevo
+MailerSend
+Mailchimp
```

Comment:
you also probably want to merge in `main` because the location of `accept.txt` moved due to this PR: https://github.com/FusionAuth/fusionauth-site/pull/3178

Reply:
totally missed this as well, @sixhobbits if you could merge and update these in
`config/vale/styles/config/vocabularies/FusionAuth/accept.txt`

Reply:
@sixhobbits looks like after main merge vale has moved...

`config/vale/styles/config/vocabularies/FusionAuth/accept.txt`